obstacle_rich_env/configs/lidar_configs.py

```diff
@@ -0,0 +1,14 @@
+from attrdict import AttrDict as AD
+import torch
+
+lidar_configs = AD(max_range=5,
+                   ray_num=100,
+                   scan_angle=[-torch.pi, torch.pi],
+                   update_rate=0.2,
+                   ray_sampling_rate=1000,
+                   space_dimension=2,
+                   sensor_mounting_location=[0, 0],
```

Comment:
Can we do XYZ, RPY instead?

Reply:
For this version, I would like to keep it as 2D. In the future, I will update the LiDAR class to a more general version to cover both 3D and 2D environments. Variable names have been changed as you suggested.